code/modules/organs/blood.dm

```diff
@@ -128,16 +128,10 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 		for(var/obj/item/organ/external/temp in organs)
 			if(!(temp.status & ORGAN_BLEEDING) || temp.status & ORGAN_ROBOT)
 				continue
-			if(isvaurca(src))
-				for(var/datum/wound/W in temp.wounds) if(W.bleeding())
-					blood_max += W.damage / 30
-				if (temp.open)
-					blood_max += 3  //Yer stomach is cut open
-			else
-				for(var/datum/wound/W in temp.wounds) if(W.bleeding())
-					blood_max += W.damage / 40
-				if (temp.open)
-					blood_max += 2  //Yer stomach is cut open
+			for(var/datum/wound/W in temp.wounds) if(W.bleeding())
+				blood_max += ((W.damage / 40) * species.fall_mod)
```

Comment:
This represents a 3x greater bloodloss from previous values for Vaurca (who already had a 30% increase to bloodloss, meaning that this is a 330% increased bloodloss for Vaurca)